rho/translation.py

```diff
@@ -19,6 +19,7 @@
 def get_translation():
     """Obtains the locale based translation setup for rho"""
     if hasattr(T, 'ugettext'):
+        # pylint: disable=no-member
```

Comment:
That would work, but I would find it less clear to use the `getattr` method than the if statement.